{"uid":"e7fc9329dd2d9ed5","name":"Test Delete Admin Agent","fullName":"test_cases.test_agent.TestAgentManagement#test_delete_admin_agent","historyId":"bf5102b0fa26be747eb8ee000c1d24e7","time":{"start":1726461465537,"stop":1726461465537,"duration":0},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"autonomous_agent_api","time":{"start":1726460968428,"stop":1726460968428,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_response","time":{"start":1726460969028,"stop":1726460969152,"duration":124},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_cookie","time":{"start":1726460969152,"stop":1726460969152,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_template_fixture","time":{"start":1726460969152,"stop":1726460969283,"duration":131},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_template_fixture","time":{"start":1726460969283,"stop":1726460969429,"duration":146},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_admin_agent_fixture","time":{"start":1726460969429,"stop":1726460969568,"duration":139},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_admin_agent_fixture","time":{"start":1726460969901,"stop":1726460970053,"duration":152},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"load_funds_to_agent_address","time":{"start":1726460970055,"stop":1726461091752,"duration":121697},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"run_admin_agent_fixture","time":{"start":1726461091752,"stop":1726461464986,"duration":373234},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_admin_agent_fixture","time":{"start":1726461464986,"stop":1726461465536,"duration":550},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[],"attachments":[{"uid":"e965367b6283ac8","name":"log","source":"e965367b6283ac8.txt","type":"text/plain","size":305},{"uid":"34a7e0940a16174c","name":"stdout","source":"34a7e0940a16174c.txt","type":"text/plain","size":451},{"uid":"64c9b790f5048cbe","name":"stderr","source":"64c9b790f5048cbe.txt","type":"text/plain","size":1137}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":3,"hasContent":true},"afterStages":[{"name":"autonomous_agent_api::0","time":{"start":1726461467155,"stop":1726461467155,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"parentSuite","value":"Agent Management"},{"name":"feature","value":"Delete Admin Agent"},{"name":"suite","value":"test_agent"},{"name":"subSuite","value":"TestAgentManagement"},{"name":"host","value":"fv-az659-915"},{"name":"thread","value":"1923-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_cases.test_agent"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":6,"broken":0,"skipped":0,"passed":15,"unknown":0,"total":21},"items":[{"uid":"627e4c8c897ce279","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/627e4c8c897ce279","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110333,"stop":1726056110333,"duration":0}},{"uid":"28cf64adad376add","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/28cf64adad376add","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674581,"stop":1726051674581,"duration":0}},{"uid":"1f880bfce7aeabef","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/1f880bfce7aeabef","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884375,"stop":1726047884375,"duration":0}},{"uid":"a3fadd300fb153c1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/a3fadd300fb153c1","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915625,"stop":1726037915625,"duration":0}},{"uid":"160f6b75b17a2ecd","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/15//#testresult/160f6b75b17a2ecd","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/b9e0c627-e84e-49c9-b257-7595ede11aa7 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726031419944,"stop":1726031419944,"duration":0}},{"uid":"77598ddf2a135c1b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/14//#testresult/77598ddf2a135c1b","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/32a5ddcb-4aec-4cae-898e-9ca7701cf69b to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1725963147569,"stop":1725963147569,"duration":0}},{"uid":"9800655b934710ec","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/13//#testresult/9800655b934710ec","status":"passed","time":{"start":1725950966365,"stop":1725950966365,"duration":0}},{"uid":"7e5638783201c93c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/12//#testresult/7e5638783201c93c","status":"passed","time":{"start":1725869818071,"stop":1725869818071,"duration":0}},{"uid":"23c4051d4050c4c8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/11//#testresult/23c4051d4050c4c8","status":"passed","time":{"start":1725616132260,"stop":1725616132260,"duration":0}},{"uid":"b2701ca6aba5b25c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/10//#testresult/b2701ca6aba5b25c","status":"passed","time":{"start":1725615363211,"stop":1725615363212,"duration":1}},{"uid":"b43cd2ba4caa8db4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/9//#testresult/b43cd2ba4caa8db4","status":"passed","time":{"start":1725615331635,"stop":1725615331635,"duration":0}},{"uid":"b5c4ac90558fc1c6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/8//#testresult/b5c4ac90558fc1c6","status":"passed","time":{"start":1724135314803,"stop":1724135314803,"duration":0}},{"uid":"d4617425f3f85523","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/7//#testresult/d4617425f3f85523","status":"passed","time":{"start":1724134686544,"stop":1724134686544,"duration":0}},{"uid":"299a78dcc5610464","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/6//#testresult/299a78dcc5610464","status":"passed","time":{"start":1724133854500,"stop":1724133854500,"duration":0}},{"uid":"60163e1aacb816aa","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/5//#testresult/60163e1aacb816aa","status":"passed","time":{"start":1724132698253,"stop":1724132698253,"duration":0}},{"uid":"187e94e9888ed20","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/4//#testresult/187e94e9888ed20","status":"passed","time":{"start":1723720438134,"stop":1723720438134,"duration":0}},{"uid":"6e13f48b93d5ee1c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/3//#testresult/6e13f48b93d5ee1c","status":"passed","time":{"start":1723719852956,"stop":1723719852957,"duration":1}},{"uid":"aa92439666dfa402","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/2//#testresult/aa92439666dfa402","status":"passed","time":{"start":1723717604834,"stop":1723717604834,"duration":0}},{"uid":"f9fd7fd7f7960f5c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/1//#testresult/f9fd7fd7f7960f5c","status":"passed","time":{"start":1723716845600,"stop":1723716845601,"duration":1}},{"uid":"d92b23d2b109580d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/1//#testresult/d92b23d2b109580d","status":"passed","time":{"start":1723716209362,"stop":1723716209362,"duration":0}}]},"tags":[]},"source":"e7fc9329dd2d9ed5.json","parameterValues":[]}